pkgs/applications/networking/remote/citrix-workspace/sources.nix

```diff
@@ -87,6 +87,17 @@ let
       x86hash   = "f2dc1fd64e5314b62ba87f384958c2bbd48b06b55bed10345cddb05fdc8cffa1";
       x64suffix = "16";
       x86suffix = "16";
+      homepage  = "https://www.citrix.com/downloads/workspace-app/linux/workspace-app-for-linux/workspace-app-for-linux-2205.html";
+    };
+
+    "22.07.0" = {
+      major     = "22";
+      minor     = "7";
+      patch     = "0";
+      x64hash   = "1xf5hqshbzvl60msidnkl8xkjn17p86vxdwzph64pb1ymmw48zm1";
```

Comment:
Previously we always used the value from the Citrix website which has a different format, e.g. in this case a17e4478ad3eac4b0cbc9fb7be0dba2758393ba2d3b6a82b3074ff053586c5f5